# -----------------------------------------------------------------------------------
# variable names used in dmp_trajectory.cpp
# -----------------------------------------------------------------------------------

trajectory:

  controller_left_arm: l_arm_x l_arm_xd l_arm_xdd l_arm_y l_arm_yd l_arm_ydd l_arm_z l_arm_zd l_arm_zdd l_arm_roll l_arm_rolld l_arm_rolldd l_arm_pitch l_arm_pitchd l_arm_pitchdd l_arm_yaw l_arm_yawd l_arm_yawdd l_shoulder_pan l_shoulder_pan_d l_shoulder_pan_dd l_shoulder_lift l_shoulder_lift_d l_shoulder_lift_dd l_upper_arm_roll l_upper_arm_roll_d l_upper_arm_roll_dd l_elbow_flex l_elbow_flex_d l_elbow_flex_dd l_forearm_roll l_forearm_roll_d l_forearm_roll_dd l_wrist_flex l_wrist_flex_d l_wrist_flex_dd l_wrist_roll l_wrist_roll_d l_wrist_roll_dd
  cart_and_joint_left_arm:  l_arm_x l_arm_xd l_arm_xdd l_arm_y l_arm_yd l_arm_ydd l_arm_z l_arm_zd l_arm_zdd l_arm_q0 l_arm_q0d l_arm_q0dd l_arm_q1 l_arm_q1d l_arm_q1dd l_arm_q2 l_arm_q2d l_arm_q2dd l_arm_q3 l_arm_q3d l_arm_q3dd  l_shoulder_pan l_shoulder_pan_d l_shoulder_pan_dd l_shoulder_lift l_shoulder_lift_d l_shoulder_lift_dd l_upper_arm_roll l_upper_arm_roll_d l_upper_arm_roll_dd l_elbow_flex l_elbow_flex_d l_elbow_flex_dd l_forearm_roll l_forearm_roll_d l_forearm_roll_dd l_wrist_flex l_wrist_flex_d l_wrist_flex_dd l_wrist_roll l_wrist_roll_d l_wrist_roll_dd

  debug_cart_left_arm: time dt unspecified l_arm_x l_arm_xd l_arm_xdd l_arm_y l_arm_yd l_arm_ydd l_arm_z l_arm_zd l_arm_zdd l_arm_q0 l_arm_q0d l_arm_q0dd l_arm_q1 l_arm_q1d l_arm_q1dd l_arm_q2 l_arm_q2d l_arm_q2dd l_arm_q3 l_arm_q3d l_arm_q3dd
  cart_left_arm: l_arm_x l_arm_xd l_arm_xdd l_arm_y l_arm_yd l_arm_ydd l_arm_z l_arm_zd l_arm_zdd l_arm_q0 l_arm_q0d l_arm_q0dd l_arm_q1 l_arm_q1d l_arm_q1dd l_arm_q2 l_arm_q2d l_arm_q2dd l_arm_q3 l_arm_q3d l_arm_q3dd
  l_arm_x:
    name: l_arm_x
    unit: m
  l_arm_xd:
    name: l_arm_xd
    unit: m/s
  l_arm_xdd:
    name: l_arm_xdd
    unit: m/s^2
  l_arm_y:
    name: l_arm_y
    unit: m
  l_arm_yd:
    name: l_arm_yd
    unit: m/s
  l_arm_ydd:
    name: l_arm_ydd
    unit: m/s^2
  l_arm_z:
    name: l_arm_z
    unit: m
  l_arm_zd:
    name: l_arm_zd
    unit: m/s
  l_arm_zdd:
    name: l_arm_zdd
    unit: m/s^2
  l_arm_q0:
    name: l_arm_q0
    unit: "-"
  l_arm_q0d:
    name: l_arm_q0d
    unit: "-"
  l_arm_q0dd:
    name: l_arm_q0dd
    unit: "-"
  l_arm_q1:
    name: l_arm_q1
    unit: "-"
  l_arm_q1d:
    name: l_arm_q1d
    unit: "-"
  l_arm_q1dd:
    name: l_arm_q1dd
    unit: "-"
  l_arm_q2:
    name: l_arm_q2
    unit: "-"
  l_arm_q2d:
    name: l_arm_q2d
    unit: "-"
  l_arm_q2dd:
    name: l_arm_q2dd
    unit: "-"
  l_arm_q3:
    name: l_arm_q3
    unit: "-"
  l_arm_q3d:
    name: l_arm_q3d
    unit: "-"
  l_arm_q3dd:
    name: l_arm_q3dd
    unit: "-"

  l_arm_roll:
    name: l_arm_roll
    unit: rad
  l_arm_rolld:
    name: l_arm_rolld
    unit: rad/s
  l_arm_rolldd:
    name: l_arm_rolldd
    unit: rad/s^2  
  l_arm_pitch:
    name: l_arm_pitch
    unit: rad
  l_arm_pitchd:
    name: l_arm_pitchd
    unit: rad/s
  l_arm_pitchdd:
    name: l_arm_pitchdd
    unit: rad/s^2  
  l_arm_yaw:
    name: l_arm_yaw
    unit: rad
  l_arm_yawd:
    name: l_arm_yawd
    unit: rad/s
  l_arm_yawdd:
    name: l_arm_yawdd
    unit: rad/s^2      
  
  debug_pr2_l_arm_joint_names: time dt unspecified l_shoulder_pan l_shoulder_pan_d l_shoulder_pan_dd l_shoulder_lift l_shoulder_lift_d l_shoulder_lift_dd l_upper_arm_roll l_upper_arm_roll_d l_upper_arm_roll_dd l_elbow_flex l_elbow_flex_d l_elbow_flex_dd l_forearm_roll l_forearm_roll_d l_forearm_roll_dd l_wrist_flex l_wrist_flex_d l_wrist_flex_dd l_wrist_roll l_wrist_roll_d l_wrist_roll_dd
  pr2_l_arm_joint_names: l_shoulder_pan l_shoulder_pan_d l_shoulder_pan_dd l_shoulder_lift l_shoulder_lift_d l_shoulder_lift_dd l_upper_arm_roll l_upper_arm_roll_d l_upper_arm_roll_dd l_elbow_flex l_elbow_flex_d l_elbow_flex_dd l_forearm_roll l_forearm_roll_d l_forearm_roll_dd l_wrist_flex l_wrist_flex_d l_wrist_flex_dd l_wrist_roll l_wrist_roll_d l_wrist_roll_dd
  l_shoulder_pan:
    name: l_shoulder_pan
    unit: rad
  l_shoulder_pan_d:
    name: l_shoulder_pan_d
    unit: rad/s
  l_shoulder_pan_dd:
    name: l_shoulder_pan_dd
    unit: rad/s^2
  l_shoulder_lift:
    name: l_shoulder_lift
    unit: rad
  l_shoulder_lift_d:
    name: l_shoulder_lift_d
    unit: rad/s
  l_shoulder_lift_dd:
    name: l_shoulder_lift_dd
    unit: rad/s^2
  l_upper_arm_roll:
    name: l_upper_arm_roll
    unit: rad
  l_upper_arm_roll_d:
    name: l_upper_arm_roll_d
    unit: rad/s
  l_upper_arm_roll_dd:
    name: l_upper_arm_roll_dd
    unit: rad/s^2
  l_elbow_flex:
    name: l_elbow_flex
    unit: rad
  l_elbow_flex_d:
    name: l_elbow_flex_d
    unit: rad/s
  l_elbow_flex_dd:
    name: l_elbow_flex_dd
    unit: rad/s^2
  l_forearm_roll:
    name: l_forearm_roll
    unit: rad
  l_forearm_roll_d:
    name: l_forearm_roll_d
    unit: rad/s
  l_forearm_roll_dd:
    name: l_forearm_roll_dd
    unit: rad/s^2
  l_wrist_flex:
    name: l_wrist_flex
    unit: rad
  l_wrist_flex_d:
    name: l_wrist_flex_d
    unit: rad/s
  l_wrist_flex_dd:
    name: l_wrist_flex_dd
    unit: rad/s^2
  l_wrist_roll:
    name: l_wrist_roll
    unit: rad
  l_wrist_roll_d:
    name: l_wrist_roll_d
    unit: rad/s
  l_wrist_roll_dd:
    name: l_wrist_roll_dd
    unit: rad/s^2
